### PR TITLE
Math Magicians Components

### PR DESCRIPTION
**Pull request by BHObserver**
## In this branch >>

- **Create Component for calculator app**
- **Design the UI of calculator as the requirements**

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        </span>
+        <span className="btn-container-five">
+          <button type="button" className="zero">0</button>
+          <button type="button">.</button>
+          <button type="button" className="bg-color">=</button>
+        </span>
+      </div>
+    </div>
+  );
+}
```
Comment (DammyShittu) [inline src/components/calculator.js]: - [ ] The Calculator component should have at least one child component. Kindly add this component.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @BHObserver,

## Highlights
✔️ Immaculate calculate design

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (levy002) [inline src/components/calculator.js]: - [ ] As requested by the previous reviewer and in order to align with the project requirements, it's advisable to incorporate at least one child component within your Calculator react component. This not only enhances code organization but also contributes to better maintainability.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @BHObserver   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (BHObserver) [reply src/components/calculator.js]: @levy002  Sorry about that. I didn't push the changes.
Could you please check it again, I just want to be sure if my changes were correct.

**Review by Alusp — APPROVED** [review]
Hi @BHObserver, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
congrats

 ## Highlights
✅ Implemented a component for Calculator
✅ Linters are passing
✅ Well descriptive PR
✅ Use React components.
✅ Implement the design only (JSX and CSS)
✅ Create at least 1 child component

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**